Makefile

```
@@ -112,10 +112,12 @@ generate: controller-gen
 # Build a single-architecture docker image
 docker-build: test
 	DOCKER_BUILDKIT=1 docker build -t ${IMG} .
+	docker build -t ${ORG}/grafana-plugins-init:${TAG} ./grafana_plugins_init
```

Maybe enable buildkit for consistency
```suggestion
	DOCKER_BUILDKIT=1 docker build -t ${ORG}/grafana-plugins-init:${TAG} ./grafana_plugins_init
```